Client/Frontend/Browser/Tab.swift

```diff
@@ -151,6 +159,9 @@ class Tab: NSObject {
             configuration!.userContentController = WKUserContentController()
             configuration!.preferences = WKPreferences()
             configuration!.preferences.javaScriptCanOpenWindowsAutomatically = false
+            // Add userScript to configuration if necessary
+            let userScript = getUserScript()
+            configuration!.userContentController.addUserScript(userScript!)
```

Comment:
The pattern we use here is to use add a `TabHelper` to the `Tab` in `BrowserViewController.tab(tab: Tab, didCreateWebView webView: WKWebView)`.

The tab helper looks after the adding of the user script.